cdk/README.md

```diff
@@ -30,16 +30,15 @@ cd resources/inventoryLambda; npm install
 If it's the first time CDK use in an environment, the environment needs to be [prepared](https://docs.aws.amazon.com/cdk/v2/guide/bootstrapping.html) for the deployment. Run the following first
 
 ```
-cdk bootstrap aws://AWS_ACCOUNT_NUMBER/REGION [--profile AWS_PROFILE] --context env=dev
+cd PROJECT_ROOT/cdk; cdk bootstrap aws://AWS_ACCOUNT_NUMBER/REGION [--profile AWS_PROFILE] --context env=dev
```

Comment:
Do you think it may be a good idea to say (remove square brackets). this drove me insane! Another layer of possible mistake checking to confirm when being run.

Reply:
It is a fairly standard way of representing optional parameters on a command line - take a look at pretty much any manpage for a bash command :)

Reply:
But then so is..... <don't use the brackets> at the very least an example? I'm not sure you should rely on others having the same amount of experience as oneself.

Reply:
Meh - there is (and should be) a minimum level of experience in this type of documentation. The syntax patterns exist for a reason. But I'll add an extra clause without the profile for clarity here

Reply:
actually the explanation is already there: 
`
Here `aws://AWS_ACCOUNT_NUMBER/REGION` is the AWS account and region to use for the deployment, e.g. `aws://1234567890/eu-west-2`.
Use `--profile AWS_PROFILE` if necessary to choose the correct [AWS CLI access keys](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html).
`

Reply:
So it is my bad!